tools/scripts/generic.mk

```diff
@@ -221,6 +221,7 @@ endif
 #Add to include all directories containing a .h file
 EXTRA_INC_PATHS += $(sort $(foreach dir, $(INCS_IN_BUILD),$(dir $(dir))))
 CFLAGS += $(addprefix -I,$(EXTRA_INC_PATHS) $(PLATFORM_INCS))
+CFLAGS += $(VERB_INC_FLAGS)
```

Comment:
nitpick: I think is cleaner if you just add -I to all PLATFORM_INCS and 
CFLAGS += $(PLATFORM_INCS)